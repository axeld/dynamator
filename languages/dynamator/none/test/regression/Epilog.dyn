<dynamator language="none">
<epilog>
<!--RETAIN This is the end. RETAIN-->
</epilog>
</dynamator>


