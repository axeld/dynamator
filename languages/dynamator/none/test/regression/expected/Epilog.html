<html>
  <head>
    <title>Empty HTML file</title>
  </head>
  <body>
  </body>
</html>

<!--RETAIN This is the end. RETAIN-->
<!-- generated by Dynamator Sat Jul 26 06:53:52 CDT 2003 -->
